packages/strapi/lib/utils/index.js

```diff
@@ -14,16 +14,10 @@ const exposer = require('./exposer');
 
 module.exports = {
   loadFile: function(url) {
-    try {
-      // Clear cache.
-      delete require.cache[require.resolve(path.resolve(this.config.appPath, url))];
-      // Require without cache.
-      return require(path.resolve(this.config.appPath, url));
-    } catch (e) {
-      this.log.error(e);
-
-      return {};
-    }
+    // Clear cache.
+    delete require.cache[require.resolve(path.resolve(this.config.appPath, url))];
+    // Require without cache.
+    return require(path.resolve(this.config.appPath, url));
```

Comment:
I supposed this `require` is catch when this function is called ?

Reply:
No, it was a bad practice because when there was a typo in a file, the error wasn't thrown and the server was starting without any issue which is bad.